connector/src/main/java/org/jboss/as/connector/logging/ConnectorLogger.java

```diff
@@ -648,22 +648,22 @@ public interface ConnectorLogger extends BasicLogger {
     @Message(id = 69, value = "At least one xa-datasource-property is required for an xa-datasource")
     OperationFailedException xaDataSourcePropertiesNotPresent();
 
-    /**
-     * A message indicating that jndi-name is missing and it's a required attribute
-     *
-     * @return the message.
-     */
-    @Message(id = 70, value = "Jndi name is required")
-    OperationFailedException jndiNameRequired();
+//    /**
+//     * A message indicating that jndi-name is missing and it's a required attribute
+//     *
+//     * @return the message.
+//     */
+//    @Message(id = 70, value = "Jndi name is required")
+//    OperationFailedException jndiNameRequired();
```

Comment:
I think that's an IDE issue, commenting the comment

Reply:
@iweiss This is what we normally do. If you don't comment out the javadoc, just the normal code, you get IDE warns.